yarn.lock

```diff
@@ -2,33 +2,25 @@
 # yarn lockfile v1
 
 
-"@ampproject/remapping@2.2.1":
+"@ampproject/remapping@2.2.1", "@ampproject/remapping@^2.1.0", "@ampproject/remapping@^2.2.0":
```

Comment:
Can you please revert these changes.
The rest is looking good, I'm trying to go through #3865 to verify the output of this PR.